### DISCUSSION TITLE
Prediction of RTs and spectra for non-tryptic peptides

### DISCUSSION THREAD

**Discussion by diabeginner01** [vdemichev/DiaNN#588] [Q&A]
Hi,
I have a FASTA of peptides (about 20-30 aa long) that are non-tryptic nor do they have sequences that adhere to the other enzyme specificities that DIA-NN offers for digestion. Would DIA-NN reasonably predict their RTs and spectra in the library free mode or is this non-feasible and prone to errors?

**Comment by vdemichev** [comment]
Yes, prediction will work fine.
